root/release/edit/layout.tt

```diff
@@ -64,7 +64,7 @@
         <button type="button" data-click="open">[% l('Add Medium') %]</button>
```

Comment:
“Add Medium” should probably be put on the left of “Previous” for consistency with “Enter edit” button.

Reply:
Not sure I agree, because while you pretty much always want to enter the edit on the last tab, you don't always (or even often) want to add a new medium on the tracklist tab. That said, I'm not opposed to ordering it like that if you prefer either :)